<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Light Defender</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../face_detec/resources/bg4.jpg</normaloff>../face_detec/resources/bg4.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#detectionPage{
	background-image: url(resources/system_bg);
}
#loginPage{
	background-image: url(resources/bg4);
}
#settingsPage{
	background-image: url(resources/bg4);
}
#facePage{
	background-image: url(resources/bg4);
}
QPushButton{
		
font: 20pt &quot;Agency FB&quot;;
  padding: 0;

  overflow: hidden;

  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  
  background-color: rgba(78, 78, 78,0.9);
  color: #ecf0f1;
  
  transition: background-color .3s;

}
QPushButton:hover{
	background-color: rgba(108,108,108,1);
}
QPushButton:pressed{
	background-color: rgba(48,48,48,1);
	color: rgb(255,255,255);
	border: 1px solid rgb(78,78,78);
}
QLabel#faceCoords, #laserCoords, #faceLabel, #laserLabel{
	color: rgb(200,200,200);
	font: 75 14pt &quot;Agency FB&quot;;
}
</string>
   </property>
   <widget class="QStackedWidget" name="sceneManager">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginPage">
     <property name="styleSheet">
      <string notr="true">QLabel{
	font: 75 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}

QLineEdit{
	background-color: transparent;
	border: 0px solid black;
	border-bottom: 2px solid rgb(200,200,200);
	outline:none;
	font: 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QLineEdit:focus{
	border-color:dodgerBlue;
    box-shadow:0 0 8px 0 dodgerBlue;
}</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>User Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="userText">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="passText">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>90</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Agency FB&quot;;
color: rgb(180,180,180);</string>
      </property>
      <property name="text">
       <string>Welcome to Light Defender</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginButton{
	
	font: 20pt &quot;Agency FB&quot;;
  padding: 0;

  overflow: hidden;

  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  
  background-color: rgba(48, 48, 48,0.9);
  color: #ecf0f1;
  
  transition: background-color .3s;
}
#loginButton:hover{
	background-color: rgba(18, 18, 18,0.9);
}
#loginButton:pressed{
background-color: rgba(236, 240, 241, .3);
}
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginMessage">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>390</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(230,10,10);
font-weight: bold</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsPage">
     <property name="styleSheet">
      <string notr="true">QLabel{
	font: 75 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QCheckBox{
font: 75 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QRadioButton{
	font: 75 20pt &quot;Agency FB&quot;;
	font-weight: bold;
	color: rgb(50,50,50);
}

QLineEdit{
	background-color: transparent;
	border: 0px solid black;
	border-bottom: 2px solid rgb(200,200,200);
	outline:none;
	font: 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QLineEdit:focus{
	border-color:dodgerBlue;
    box-shadow:0 0 8px 0 dodgerBlue;
}
QPushButton{
	
	font: 20pt &quot;Agency FB&quot;;
  padding: 0;

  overflow: hidden;

  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  
  background-color: rgba(48, 48, 48,0.9);
  color: #ecf0f1;
  
  transition: background-color .3s;
}
QPushButton:hover{
	background-color: rgba(18, 18, 18,0.9);
}
QPushButton:pressed{
background-color: rgba(236, 240, 241, .3);
}
</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>341</width>
        <height>290</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Phone Number:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="phoneText">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Notification Interval:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="intervalText">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="notificationBox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Enable Notifications</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="muteBox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Enable Sounds</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Laser Control:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="autoRadio">
         <property name="text">
          <string>Auto</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="manualRadio">
         <property name="text">
          <string>Manual</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Agency FB&quot;;
color: rgb(180,180,180);</string>
      </property>
      <property name="text">
       <string>User Settings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>480</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginButton{
	
	font: 20pt &quot;Agency FB&quot;;
  padding: 0;

  overflow: hidden;

  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  
  background-color: rgba(48, 48, 48,0.9);
  color: #ecf0f1;
  
  transition: background-color .3s;
}
#loginButton:hover{
	background-color: rgba(18, 18, 18,0.9);
}
#loginButton:pressed{
background-color: rgba(236, 240, 241, .3);
}
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>480</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginButton{
	
	font: 20pt &quot;Agency FB&quot;;
  padding: 0;

  overflow: hidden;

  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  
  background-color: rgba(48, 48, 48,0.9);
  color: #ecf0f1;
  
  transition: background-color .3s;
}
#loginButton:hover{
	background-color: rgba(18, 18, 18,0.9);
}
#loginButton:pressed{
background-color: rgba(236, 240, 241, .3);
}
</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorMessage">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>530</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(230,10,10);
font-weight: bold</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="facePage">
     <property name="styleSheet">
      <string notr="true">QLabel{
	font: 75 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QCheckBox{
font: 75 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QRadioButton{
	font: 75 20pt &quot;Agency FB&quot;;
	font-weight: bold;
	color: rgb(50,50,50);
}

QLineEdit{
	background-color: transparent;
	border: 0px solid black;
	border-bottom: 2px solid rgb(200,200,200);
	outline:none;
	font: 20pt &quot;Agency FB&quot;;
	color: rgb(230,230,230);
}
QLineEdit:focus{
	border-color:dodgerBlue;
    box-shadow:0 0 8px 0 dodgerBlue;
}
QPushButton{
	
	font: 20pt &quot;Agency FB&quot;;
  padding: 0;

  overflow: hidden;

  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  
  background-color: rgba(48, 48, 48,0.9);
  color: #ecf0f1;
  
  transition: background-color .3s;
}
QPushButton:hover{
	background-color: rgba(18, 18, 18,0.9);
}
QPushButton:pressed{
background-color: rgba(236, 240, 241, .3);
}
</string>
     </property>
    </widget>
    <widget class="QWidget" name="detectionPage">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>644</width>
        <height>562</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="imageLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>640</width>
             <height>480</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../face_detec/resources/bg3.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="faceLabel">
           <property name="text">
            <string>Face coordinates:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="faceCoords">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="laserLabel">
           <property name="text">
            <string>Laser coordinates:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="laserCoords">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
